### ISSUE TITLE
Keyboard focus is not visible on table controls present in "Filter records" tab.

### ISSUE THREAD

**Issue by msftedad** [gradio-app/gradio#11811]
### Describe the bug

**Test Environment:**  

OS: Windows 11[Version 24H2(OS Build 24631.3593)]
Edge version: 133.0.3065.59 (Official build) (64-bit)
URL: [Gradio Playground](https://www.gradio.app/playground?demo=Filter_Records&code=aW1wb3J0IGdyYWRpbyBhcyBncgoKZGVmIGZpbHRlcl9yZWNvcmRzKHJlY29yZHMsIGdlbmRlcik6CiAgICByZXR1cm4gcmVjb3Jkc1tyZWNvcmRzWyJnZW5kZXIiXSA9PSBnZW5kZXJdCgpkZW1vID0gZ3IuSW50ZXJmYWNlKAogICAgZmlsdGVyX3JlY29yZHMsCiAgICBbCiAgICAgICAgZ3IuRGF0YWZyYW1lKAogICAgICAgICAgICBoZWFkZXJzPVsibmFtZSIsICJhZ2UiLCAiZ2VuZGVyIl0sCiAgICAgICAgICAgIGRhdGF0eXBlPVsic3RyIiwgIm51bWJlciIsICJzdHIiXSwKICAgICAgICAgICAgcm93X2NvdW50PSgzLCAiZHluYW1pYyIpLAogICAgICAgICAgICBjb2xfY291bnQ9KDMsICJkeW5hbWljIikgLAogICAgICAgICksCiAgICAgICAgZ3IuRHJvcGRvd24oWyJNIiwgIkYiLCAiTyJdKSwKICAgIF0sCiAgICAiZGF0YWZyYW1lIiwKICAgIGRlc2NyaXB0aW9uPSJFbnRlciBnZW5kZXIgYXMgJ00nLCAnRicsIG9yICdPJyBmb3Igb3RoZXIuIiwKKQoKaWYgX19uYW1lX18gPT0gIl9fbWFpbl9fIjoKICAgIGRlbW8ubGF1bmNoKCkK&reqs=)

**User Impact:**
If there is no visual indication of focus, it will disrupt a keyboard user as well as users with attention limitation's ability to visually determine the component on which keyboard operations will interact at any point in time. 

**Repro Steps:**
1. Open URL: [Gradio Playground](https://www.gradio.app/playground?demo=Filter_Records&code=aW1wb3J0IGdyYWRpbyBhcyBncgoKZGVmIGZpbHRlcl9yZWNvcmRzKHJlY29yZHMsIGdlbmRlcik6CiAgICByZXR1cm4gcmVjb3Jkc1tyZWNvcmRzWyJnZW5kZXIiXSA9PSBnZW5kZXJdCgpkZW1vID0gZ3IuSW50ZXJmYWNlKAogICAgZmlsdGVyX3JlY29yZHMsCiAgICBbCiAgICAgICAgZ3IuRGF0YWZyYW1lKAogICAgICAgICAgICBoZWFkZXJzPVsibmFtZSIsICJhZ2UiLCAiZ2VuZGVyIl0sCiAgICAgICAgICAgIGRhdGF0eXBlPVsic3RyIiwgIm51bWJlciIsICJzdHIiXSwKICAgICAgICAgICAgcm93X2NvdW50PSgzLCAiZHluYW1pYyIpLAogICAgICAgICAgICBjb2xfY291bnQ9KDMsICJkeW5hbWljIikgLAogICAgICAgICksCiAgICAgICAgZ3IuRHJvcGRvd24oWyJNIiwgIkYiLCAiTyJdKSwKICAgIF0sCiAgICAiZGF0YWZyYW1lIiwKICAgIGRlc2NyaXB0aW9uPSJFbnRlciBnZW5kZXIgYXMgJ00nLCAnRicsIG9yICdPJyBmb3Igb3RoZXIuIiwKKQoKaWYgX19uYW1lX18gPT0gIl9fbWFpbl9fIjoKICAgIGRlbW8ubGF1bmNoKCkK&reqs=) i Edge browser.
2. Press tab key to navigate to the Tables present in the page.
3. Observe whether Keyboard focus is visible on tables or not.

**Actual Results:**
Keyboard focus is not visible on tables present in "Filter records" tab.

**Expected Results:**
Keyboard focus should be clearly visible on tables present in "Filter records" tab.

### Have you searched existing issues?  🔎

- [x] I have searched and found no existing issues

### Reproduction

https://www.gradio.app/playground?demo=Filter_Records&code=aW1wb3J0IGdyYWRpbyBhcyBncgoKZGVmIGZpbHRlcl9yZWNvcmRzKHJlY29yZHMsIGdlbmRlcik6CiAgICByZXR1cm4gcmVjb3Jkc1tyZWNvcmRzWyJnZW5kZXIiXSA9PSBnZW5kZXJdCgpkZW1vID0gZ3IuSW50ZXJmYWNlKAogICAgZmlsdGVyX3JlY29yZHMsCiAgICBbCiAgICAgICAgZ3IuRGF0YWZyYW1lKAogICAgICAgICAgICBoZWFkZXJzPVsibmFtZSIsICJhZ2UiLCAiZ2VuZGVyIl0sCiAgICAgICAgICAgIGRhdGF0eXBlPVsic3RyIiwgIm51bWJlciIsICJzdHIiXSwKICAgICAgICAgICAgcm93X2NvdW50PSgzLCAiZHluYW1pYyIpLAogICAgICAgICAgICBjb2xfY291bnQ9KDMsICJkeW5hbWljIikgLAogICAgICAgICksCiAgICAgICAgZ3IuRHJvcGRvd24oWyJNIiwgIkYiLCAiTyJdKSwKICAgIF0sCiAgICAiZGF0YWZyYW1lIiwKICAgIGRlc2NyaXB0aW9uPSJFbnRlciBnZW5kZXIgYXMgJ00nLCAnRicsIG9yICdPJyBmb3Igb3RoZXIuIiwKKQoKaWYgX19uYW1lX18gPT0gIl9fbWFpbl9fIjoKICAgIGRlbW8ubGF1bmNoKCkK&reqs=

### Screenshot

1. Keyboard focus is not visible on table controls present in Filter records tab.png
2. Keyboard focus is not visible on table controls present in Filter records tab.mp4

**

https://github.com/user-attachments/assets/c36bb83b-ccb9-408e-bc78-0c873410f6aa
<img width="960" height="591" alt="Image" src="https://github.com/user-attachments/assets/f1f62281-3f3a-4184-8929-bd84b6d3db86" />

**

### Logs

```shell

```

### System Info

```shell
Gradio Documentation version 5.31.0
```

### Severity

I can work around it

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by abidlabs** [comment]
Can't repro the original issue, may have been specific to gradio-lite:

<img width="1365" height="146" alt="Image" src="https://github.com/user-attachments/assets/6bb8276c-054c-4dce-bd15-f1bd30fcfa02" />
